oxidation/client/package.json

```diff
@@ -3,28 +3,22 @@
     "version": "0.0.1",
     "private": true,
     "scripts": {
-        "serve": "vue-cli-service serve",
         "postinstall": "cd electron && npm install",
         "test:unit": "vue-cli-service test:unit",
         "test:e2e": "vue-cli-service test:e2e",
         "lint": "vue-cli-service lint",
-        "build": "npx ionic build",
```

Comment:
ionic build was the same than 'vue-cli service build' no ?

Reply:
Why do we want to hide the command behind ?

Reply:
yes it was a wrapper that only launched `vue-cli service build`, hence removing it to make it more clear what is actually called